### ISSUE TITLE
Worker: game freeze with huge process on workers

### ISSUE THREAD

**Issue by fauconjona** [altmp/altv-js-module#77]
### Description of the problem

When there's some huge processes on the workers, the game freeze completely.

They're correctly working on other threads:
```
[00:51:16] generator 0 done
[00:51:16] generator 1 done
[00:51:16] generator 4 done
[00:51:16] generator 2 done
[00:51:16] generator 3 done
[00:51:30] worker 5 done
[00:51:37] worker 8 done
[00:51:46] worker 6 done
[00:51:57] worker 7 done
[00:52:09] worker 9 done
```

When workers are done, the game is no longer freeze

### Reproduction steps

worker.js
```js
alt.on('sort', (array, id) => {
    array.sort();
    alt.log('worker', id, 'done');
})

alt.on('generate', (length, id) => {
    let numbers = [];
    for (let n = 0; n < length; n++) {
        numbers.push(Math.floor(Math.random() * 500000));
    }
    alt.log('generator', id, 'done');
    alt.emit('numbers', numbers);
})
```

client.ts
```ts
const NB_WORKERS: number = 10;
let workers: any[] = [];

for (let i = 0; i < NB_WORKERS; i++) {
    workers.push(await this.createWorker(i));
}

for (let index = 0; index < workers.length / 2; index++) {
    const generatorId = index;
    const generator = workers[generatorId];
    const workerId = index + workers.length / 2;
    const worker = workers[workerId];
    
    generator.on('numbers', (numbers: number[]) => {
        worker.emit('sort', numbers, workerId);
    })
    generator.emit('generate', 5000000, generatorId)
}
```

`createWorker(i)` create a worker and start it, then a promise is resolved when it loaded

### Expected behaviour

Game not freeze and the main thread can work

### Operating system

Windows 11

### Version

dev/7.0-dev10

### Scope

client

**Comment by LeonMrBonnie** [comment]
Please test if this also happens with a single worker.

It is to be expected that all workers doing heavy work has an impact on the main thread. This is unfortunately how V8 works, so there is not really a way to work around this. (Without doing some stuff like creating a new process, but I really don't want to do that in the module)

**Comment by LeonMrBonnie** [comment]
Well, this *sounded* like a good idea but it's actually not possible from the V8 side. You can't share *any* objects across isolates, so the only way would be to make a copy, but then it would be as slow as MValues again, so there isn't really a point in doing that.
So yeah, unfortunately we have to live with this for now, maybe in the future we can try to offload the parsing to multiple ticks / another thread, but its not something thats planned currently.

**Comment by LeonMrBonnie** [comment]
This has now been fixed in 75632c04ace34f56ed6fa414c12c7a05d3020f90.
